pkg/util/config/config.go

```diff
@@ -38,6 +38,7 @@ type Resources struct {
 	NodeReadinessLabel      map[string]string `name:"node_readiness_label" default:""`
 	MaxInstances            int32             `name:"max_instances" default:"-1"`
 	MinInstances            int32             `name:"min_instances" default:"-1"`
+	ShmVolume               bool              `name:"shm_volume" default:"true"`
```

Comment:
I thing we use "enable" prefix in other config options?

enableShardedMemoryVolume ?

Reply:
Well, I actually disagree. The data type itself for this value tells that it's enable/disable option, so add it to the name would be redundant.